packages/gradle-plugin/src/main/kotlin/io/realm/kotlin/gradle/RealmPlugin.kt

```diff
@@ -47,6 +50,22 @@ open class RealmPlugin : Plugin<Project> {
         ) { /* Do nothing */ }
         getBuildEventsRegistry().onTaskCompletion(serviceProvider)
 
+        project.configurations.all { conf: Configuration ->
+            // Ensure that android unit tests uses the Realm JVM variant rather than Android.
+            if (conf.name.endsWith("UnitTestRuntimeClasspath")) {
```

Comment:
It would be nice if we could somehow use dependency attributes to only direct the 
```attribute(Attribute.of("org.jetbrains.kotlin.platform.type", String::class.java), "jvm")``` 
dependencies to the `-jvm` modules ... but I haven't been able to find a way to enforce attributes on the target dependencies.